### PR TITLE
Add default configuration for analytic vendors.

### LINKED ISSUE #7019: Pre-fill default configuration for top analytic vendors.
### Feature Description

Currently, When users select "googleanalytics" or "gtag" in Analytics section ( in AMP Settings page) It pre-fills the default configuration.

image

A similar feature, we need to implement for other top vendors. 

1. Google Analytics
2. Google Tag
3. Google Ads
4. Adobe Analytics
5. Alexa Metrics
6. Baidu analytics
7. Facebook Pixel
8. New Relic
9. Nielsen
10. Yandex Metrica

An example of default configuration can be found [here](https://github.com/ampproject/amphtml/blob/main/examples/analytics-vendors.amp.html).

### Acceptance Criteria

_No response_

### Implementation Brief

_No response_

### QA Testing Instructions

_No response_

### Demo

_No response_

### Changelog Entry

_No response_

### PR DESCRIPTION
**Pull request by dhaval-parekh**
## Summary

Fixes #7019 

Pre-fill default config when the user selects analytic vendor on the AMP setting page.

https://user-images.githubusercontent.com/8168027/162701033-ac8aa66a-8cb1-4f26-aca4-4cb016c21b73.mov

## Checklist

- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: assets/src/settings-page/analytics.js [hunk 1]
```diff
@@ -18,59 +18,10 @@ import { Button, PanelRow, BaseControl, VisuallyHidden } from '@wordpress/compon
  */
 import { Options } from '../components/options-context-provider';
 import { AMPNotice, NOTICE_SIZE_SMALL } from '../components/amp-notice';
+import vendorConfigs from './vendor-configs';
 
 const GOOGLE_ANALYTICS_VENDOR = 'googleanalytics';
```
Comment (milindmore22) [inline assets/src/settings-page/analytics.js:23]: do we still need this?

**Comment by milindmore22** [comment]
@dhaval-parekh should we add a notice with (link) for 10 vendors something like 🔗  "Learn More" pointing to the respective vendor's documentation link

Example:
image

File: assets/src/settings-page/analytics.js (hunk 1 above)
Comment (dhaval-parekh) [reply assets/src/settings-page/analytics.js:23]: Yes, This variable is used below for message.

**Comment by dhaval-parekh** [comment]
> should we add a notice with (link) for 10 vendors something like 🔗 "Learn More" pointing to the respective vendor's documentation link

Yes, We can add it.

_Let me check if i can find the documentation for those vendors._

**Review by milindmore22 — APPROVED** [review]
LGTM 👍🏼

**Comment by westonruter** [comment]
There's an E2E test failing, but I'm updating with `develop` to see if it is a temporary issue.

**Review by westonruter — CHANGES_REQUESTED** [review]
The E2E tests need to be fixed.

**Review by westonruter — APPROVED** [review]

**Pull request merged**